crates/bifrost/src/providers/replicated_loglet/tasks/seal.rs

```diff
@@ -142,7 +142,7 @@ impl SealTask {
             {
                 let mut nodeset_status = DecoratedNodeSet::from_iter(effective_nodeset);
                 nodeset_status.extend(&nodeset_checker);
-                tracing::info!(
+                debug!(
```

Comment:
I'm open to ideas on how to reduce the level of ing here, maybe we can start by making it debug, but I often find this particular line very helpful. I don't have a strong opinion.

Reply:
I was wondering how helpful this line would be for users that don't know much about the internals. Do you think that this is something we should keep on info level in case we need to debug a user setup where only info logging is enabled?

Reply:
It's a tough one, but debug is too verbose and at the moment this is a high-fidelity signal that reconfiguration is happening that I don't want to lose. As I said, I have no strong opinion, but for me, it has been a common theme for me to check this line to reason about what's happening.

Reply:
Ok, then let's keep it. Can always change it later.